app/components/case-studies-components/CaseStudyChallengesAndLearnings.tsx

```diff
@@ -1,12 +1,13 @@
 "use client";
 
-import { MainCaseStudyProps } from "@/utils/interfaces";
 import { challengeIcon, greenTick } from "@/public/svg-icons/case-study-icons";
 import { motion } from "framer-motion";
 
 import ChallengesAndLearningsCard from "./ChallengesAndLearningsCard";
+import { useCaseStudyContext } from "@/app/contexts/CaseStudyContext";
```

Comment:
Please list your imports externally first, followed by a space, and then your internal imports. 

```
// External Dependencies
import { motion } from "framer-motion";

// Internal Modules/Components
import { useCaseStudyContext } from "@/app/contexts/CaseStudyContext";
import ChallengesAndLearningsCard from "./ChallengesAndLearningsCard";
import { challengeIcon, greenTick } from "@/public/svg-icons/case-study-icons";
```